libbeat/publisher/processing/default_test.go

```diff
@@ -325,6 +325,20 @@ func TestNormalization(t *testing.T) {
 	}
 }
 
+func BenchmarkNormalization(b *testing.B) {
+	s, err := MakeDefaultSupport(true)(beat.Info{}, logp.L(), common.NewConfig())
+	require.NoError(b, err)
+
+	prog, err := s.Create(beat.ProcessingConfig{}, false)
+	require.NoError(b, err)
+
+	fields := common.MapStr{"a": "b"}
```

Comment:
Events can have different complexity, which the runtime clearly depends upon. How about:

```
cases := map[string]common.MapStr{
  "event with single field": {"a": "b"},
}

for name, event := range cases {
  b.Run(name, func(b *testing.B) {
    ...
  })
}
```

This allows us to easily add more cases now or in the future.

Reply:
makes sense. will do.